sdk/nodejs/runtime/resource.ts

```diff
@@ -177,6 +177,7 @@ export function registerResource(res: Resource, t: string, name: string, custom:
         req.setProvider(resop.providerRef);
         req.setDependenciesList(Array.from(resop.allDirectDependencyURNs));
         req.setDeletebeforereplace((<any>opts).deleteBeforeReplace || false);
+        req.setIgnorechangesList(opts.ignoreChanges || []);
```

Comment:
Can you add a test that exercises this code path in the language host?

Reply:
Done.